tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -597,6 +596,9 @@ public static UserRegistry createRegistry(SurveyMain theSm) throws SQLException
         sm = theSm;
         UserRegistry reg = new UserRegistry();
         reg.setupDB();
+        if (NORMALIZE_USER_TABLE_ORGS) {
+            reg.normalizeUserTableOrgs();
+        }
```

Comment:
Each time Survey Tool is restarted, this will run

Reply:
good